src/main/java/org/isf/usergroups/dto/GroupPermissionsDTO.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Open Hospital (www.open-hospital.org)
+ * Copyright © 2006-2024 Informatici Senza Frontiere (info@informaticisenzafrontiere.org)
+ *
+ * Open Hospital is a free and open source software for healthcare data management.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * https://www.gnu.org/licenses/gpl-3.0-standalone.html
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <https://www.gnu.org/licenses/>.
+ */
+package org.isf.usergroups.dto;
+
+import java.util.List;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+
+public record GroupPermissionsDTO(
+	@Schema(name = "permissions", example = "[48, 24]", description = "List of permissions' ids") List<Integer> permissions) {

```

Comment:
```suggestion
	@Schema(name = "permissions", example = "[48, 24]", description = "List of permissions' ids") List<Integer> permissionIds) {

```
To be more precise.

Reply:
It's enough precise with type  and the description but if you want...